Cargo.toml

```diff
@@ -180,6 +182,7 @@ pallet-evm-precompile-simple = { version = "2.0.0-dev", git = "https://github.co
 pallet-hotfix-sufficients = { version = "1.0.0", git = "https://github.com/AstarNetwork/frontier.git", branch = "polkadot-v0.9.40", default-features = false }
 
 # Golden Gate
+account-filter = { package = "substrate-account-filter", git = "https://github.com/GoldenGateGGX/ggx-frames.git", branch = "polkadot-v0.9.40", default-features = false }
```

Comment:
@akorchyn I guess would be better to change github links

Reply:
done